x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/trusted_apps_page.test.tsx

```diff
@@ -13,28 +13,18 @@ import { fireEvent } from '@testing-library/dom';
 import { MiddlewareActionSpyHelper } from '../../../../common/store/test_utils';
 import {
   ConditionEntryField,
-  GetTrustedAppsListResponse,
-  NewTrustedApp,
   OperatingSystem,
-  PostTrustedAppCreateResponse,
   TrustedApp,
 } from '../../../../../common/endpoint/types';
-import { HttpFetchOptions } from 'kibana/public';
-import {
-  TRUSTED_APPS_GET_API,
-  TRUSTED_APPS_LIST_API,
-} from '../../../../../common/endpoint/constants';
-import {
-  GetPackagePoliciesResponse,
-  PACKAGE_POLICY_API_ROUTES,
-} from '../../../../../../fleet/common';
-import { EndpointDocGenerator } from '../../../../../common/endpoint/generate_data';
+import { HttpFetchOptions, HttpFetchOptionsWithPath } from 'kibana/public';
 import { isFailedResourceState, isLoadedResourceState } from '../state';
+import { forceHTMLElementOffsetWidth } from '../../../components/effected_policy_select/test_utils';
 import { toUpdateTrustedApp } from '../../../../../common/endpoint/service/trusted_apps/to_update_trusted_app';
 import { useIsExperimentalFeatureEnabled } from '../../../../common/hooks/use_experimental_features';
-import { resolvePathVariables } from '../../../../common/utils/resolve_path_variables';
 import { licenseService } from '../../../../common/hooks/use_license';
-import { forceHTMLElementOffsetWidth } from '../../../components/effected_policy_select/test_utils';
+import { trustedAppsPageHttpApiMocks } from '../test_utils/mocks';
```

Comment:
Instead of fixing the approach that was being used in this test file for mocking APIs, I opted instead to replace all of that with the generic HTTP mocks that were created a few weeks ago for trusted apps and fleet APis. Was able to remove quite a bit of code.